packages-next/fields-document/src/DocumentEditor/Toolbar.tsx

```diff
@@ -44,6 +52,9 @@ export function Toolbar({
   viewState?: { expanded: boolean; toggle: () => void };
 }) {
   const ExpandIcon = viewState?.expanded ? Minimize2Icon : Maximize2Icon;
+  const relationship = useContext(DocumentFieldRelationshipsContext);
+  const blockComponent = useContext(ComponentBlockContext);
+  const hasBlockItems = Object.entries(relationship).length || Object.keys(blockComponent).length;
```

Comment:
this is kinda ick, since we're reaching into context and running object.entries and object.keys here *just* to check for items. 

If there's a better way I'm here for it

Reply:
This isn't really a big deal, the things in those contexts won't change at runtime while a user is using the editor so getting them isn't a problem.